persistence/sql/persister_courier.go

```diff
@@ -25,6 +25,33 @@ func (p *Persister) AddMessage(ctx context.Context, m *courier.Message) error {
 	return sqlcon.HandleError(p.GetConnection(ctx).Create(m)) // do not create eager to avoid identity injection.
 }
 
+func (p *Persister) ListMessages(ctx context.Context, filter courier.MessagesFilter) ([]courier.Message, int64, error) {
+	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.ListMessages")
+	defer span.End()
+
+	q := p.GetConnection(ctx).Where("nid=?", p.NetworkID(ctx))
```

Comment:
I think, this should be `corp.ContextualizeNID(ctx, p.nid)` to keep it consistent with the other methods in the persister.

Reply:
Yeah it was actually like that before, but after i rebased this branch onto master yesterday, i seen that `corp.ContextualizeNID(ctx, pid)` is refactored to `p.NetworkID(ctx)` in commit b0d8f3853886228a64e82437643a82b3970d6ff7. That's why i did go with `p.NetworkID(ctx)`

Reply:
Huh, interesting, I missed that—sorry about that. This looks good to me then.